core/module/Dog/dog_modules/dog_module/Shadowlamb/core/SR_Item.php

```diff
@@ -532,7 +532,7 @@ private function displayWorth()
 	 * @param int $amount
 	 * @return true|false
 	 */
-	public function useAmount(SR_Player $player, $amount=1, $modify=true)
+	public function useAmount(SR_Player $player, $amount=1, $modify=true, $use_mount=false)
```

Comment:
`$use_mount` -> `$item_from_mount' or something, perhaps?

Not very nice, but not sure how to improve other than duplicating this code. :/

Reply:
resolved with new commit